src/fstar/FStar.Interactive.CompletionTable.fs

```diff
@@ -80,13 +80,15 @@ type btree<'a> =
 | StrBranch of string * 'a * (btree<'a>) * (btree<'a>)
 (* (key: string) * (value: 'a) * (lbt: btree 'a) * (rbt: btree 'a) *)
 
-let rec btree_to_list_rev btree acc =
+let rec btree_to_list_rev (btree:btree<'a>) (acc:list<(string * 'a)>)
+  : list<(string * 'a)> =
```

Comment:
This annotation is necessary with a stricter occurs check that I experimented with. It is not actually needed in the PR, but  I kept the annotation anyway as documentation